src/main/java/org/mockito/internal/configuration/SpyAnnotationEngine.java

```diff
@@ -83,6 +83,7 @@ public AutoCloseable process(Class<?> context, Object testInstance) {
     }
 
     private static Object spyInstance(Field field, Object instance) {
+        // TODO: Should we also add an option to @Spy?
```

Comment:
Yes, but let's file a separate GitHub issue for that and link it here: https://github.com/mockito/mockito/issues/2740

Reply:
Ok. I have added the issue number to the comment.